tests/utils/test_helpers.py

```diff
@@ -147,11 +146,11 @@ def test_temporary_directory_python_3_9_or_older(mocker: MockerFixture) -> None:
     mocked_temp_dir = mocker.patch("tempfile.TemporaryDirectory")
     mocked_mkdtemp = mocker.patch("tempfile.mkdtemp")
 
-    mocked_mkdtemp.return_value = "hello from test"
+    mocked_mkdtemp.return_value = Path("hello from test")
```

Comment:
That does not make sense, since we are mocking `tempfile.mkdtemp`.

Reply:
Do you have any better idea? I am tempted to just `noqa` this and leave it as it was...

Reply:
Does it result in a warning? I wonder how ruff would decide that this string should be `Path`?

Reply:
I had a brainfart last time I guess, fixed.